pkg/roachprod/promhelperclient/promhelper_utils.go

```diff
@@ -23,7 +23,7 @@ import (
 
 var (
 	// promCredFile is where the prom helper credentials are stored
-	promCredFile = os.TempDir() + "promhelpers-secrets"
+	promCredFile = os.TempDir() + "/promhelpers-secrets"
```

Comment:
This this seems to have worked differently locally vs gceworker/CI, I think the safer approach is to do `filepath.Join(os.TempDir(), "promhelper-secrets")`.

Meta question: why use the temp dir (which could be erased on restart)? Should we use the roachprod data directory (`~/.roachprod`), or are there any downsides to that?

Reply:
This sounds good to me. There was not specific reason to use temp as such.